src/backend/database_connection.py

```diff
@@ -83,25 +77,14 @@ def get_object_feature_matrix_from_db(self, tablename, label=None, label_type=No
             obj_feature_matrix.extend(pickle.loads(image_row[1]))
         return {"images": images, "data_matrix": np.array(obj_feature_matrix)}
 
-    def get_correct_labels_for_given_images(self, image_names = None, label_type=None):
+    def get_correct_labels_for_given_images(self, image_names=None, label_type=None, tablename='metadata'):
         conn = self.get_db_connection()
         cursor = conn.cursor()
-        tablename = 'metadata'
+        # tablename = 'metadata'
         if not image_names:
-            query = "SELECT imagename, {1} FROM {0}".format(tablename, self.label_type_map.get(label_type))
+            query = "SELECT imagename, {1} FROM {0}".format(tablename, label_type)
         else:
-            query = "SELECT imagename, {1} FROM {0} WHERE imagename IN {2}".format(tablename, self.label_type_map.
-                                                                                   get(label_type), tuple(image_names))
-        cursor.execute(query)
-        result = cursor.fetchall()
-        return result
-
-    def get_metadata_for_task_8(self,image_names=None):
-        conn = self.get_db_connection()
-        cursor = conn.cursor()
-        tablename='metadata'
-        query = "SELECT imagename,gender,accessories,aspectofhand FROM {0} WHERE imagename IN {1}"\
-            .format(tablename, tuple(image_names))
+            query = "SELECT imagename, {1} FROM {0} WHERE imagename IN {2}".format(tablename, label_type, tuple(image_names))
```

Comment:
Instead of getting result in list format we should to get it in dict of panda dataframe.

Reply:
iska function to tu alag se likhne wala tha na @Thek10patil ?

Reply:
So you mean something like (hog_descriptor, label)